pkg/cmd/comp.go

```diff
@@ -90,7 +91,15 @@ func NewCompRunCommands(c types.Args, ioStreams util.IOStreams) *cobra.Command {
 func GetWorkloadNameFromArgs(args []string) (string, error) {
 	argsLength := len(args)
 	if argsLength < 1 {
-		return "", errors.New("must specify name for workload")
+		workloadList,err := InstalledWorkloads()
```

Comment:
We don't need to get workload type List here.

Reply:
The list can help the user to quickly complete the command without 1) switching to `vela workloads` and choosing a suitable one, and 2) returning back to `vela comp`.